examples/website/text/app.jsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable max-len */
+import React from 'react';
+import {useState, useCallback} from 'react';
+import {createRoot} from 'react-dom/client';
+import {Map} from 'react-map-gl';
+import maplibregl from 'maplibre-gl';
+import DeckGL from '@deck.gl/react';
+import {MapView} from '@deck.gl/core';
+import {TextLayer} from '@deck.gl/layers';
+import {CollideExtension} from '@deck.gl/extensions';
+import {scaleLinear} from 'd3-scale';
+
+import {csv} from 'd3-request';
+
+// Sample data
+const DATA_URL = 'https://raw.githubusercontent.com/visgl/deck.gl-data/master/examples/text-layer/cities-1000.csv';
```

Comment:
Performance feels worse than https://pessimistress.github.io/experiments/collide/, is it the same dataset?

Reply:
It's the same. React in development mode usually has worse perf.